theme/templates/collections/items/index.html

```diff
@@ -308,6 +308,48 @@ <h1>{% for l in data['links'] if l.rel == 'collection' %} {{ l['title'] }} {% en
             let csvLink = document.getElementById('csv-format-url')
             csvLink.href = '?' + queryParams.toString() + '&f=csv'
           }
+          const updateLangChangeLink = function() {
+            let langLink = document.getElementById('lang-switch-url')
+            if(queryParams.has('lang')){
```

Comment:
Can you ensure proper spacing? I can't recall but I think `eslint` or `prettier` is the tool to use to ensure consistent JS formatting.

```suggestion
            if (queryParams.has('lang')) {
```

Reply:
Change has been made for each `if` statement and `for` loop.